web/public/js/controllers/start-dojo-wizard-controller.js

```diff
@@ -647,6 +647,7 @@ function startDojoWizardCtrl($scope, $window, $state, $location, auth, alertServ
       var lsdl = $localStorage[currentUser.id].dojoLead.dojoListing;
 
       if(lsdl.name) $scope.dojo.name = lsdl.name;
+      if(lsdl.attendee) $scope.dojo.attendee = lsdl.attendee;
```

Comment:
If it's supposed to be a number, do the type change here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)